thirdeye-ui/src/app/components/admin-page/recent-failures/recent-failures.component.tsx

```diff
@@ -66,6 +114,30 @@ export const RecentFailures: FunctionComponent = () => {
                         </LoadingErrorStateSwitch>
                     </Typography>
                 </Grid>
+                <Grid item>
+                    <LoadingErrorStateSwitch
+                        isError={false}
+                        isLoading={status === ActionStatus.Working}
+                        loadingState={
+                            <SkeletonV1
+                                animation="pulse"
+                                height={40}
+                                width={425}
+                            />
+                        }
+                    >
```

Comment:
We can forego this nesting the time selector in this component. We should always show the time selector in this page

Reply:
Removed the loading UI